src/compas/artists/robotmodelartist.py

```diff
@@ -362,19 +373,21 @@ def update_tool(self, joint_state=None, visual=True, collision=True, transformat
 
         Parameters
         ----------
-        joint_state : :obj:`dict`or :class:`compas.robots.Configuration`, optional
+        joint_state : dictor :class:`compas.robots.Configuration`, optional
```

Comment:
this looks like a type, i think it was correct as it was 
```
        :obj:`dict` or :class:`compas.robots.Configuration`
```
optionally, if you wanna be super correct, this is a `Union[dict, compas.robots.Configuration]`, right?